### PR TITLE
[DOCS-6970] ACS 7.3 Community Updates

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Updates so far for install and zip.

### CODE REVIEW

**Review by HeinRagas — APPROVED** [review]
No comments

File: content-services/community/install/zip/tomcat.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Use this method of installing if you've already installed a JRE, a supported dat
 
 The Community Edition distribution file is a zip containing the required WAR files, in addition to the additional commands, and configuration files for a manual installation.
 
-1. Download the zip file from Alfresco Nexus repository: [alfresco-content-services-community-distribution-7.2.x.zip](https://artifacts.alfresco.com/nexus/#nexus-search;quick~alfresco-content-services-community-distribution){:target="_blank"}
+1. Download the zip file from Alfresco Nexus repository: [alfresco-content-services-community-distribution-7.3.x.zip](https://artifacts.alfresco.com/nexus/#nexus-search;quick~alfresco-content-services-community-distribution){:target="_blank"}
```
Comment (anxumalo) [inline content-services/community/install/zip/tomcat.md:104]: Looking for a better link - this search picks up an old 23.1 milestone release

Comment (anxumalo) [reply content-services/community/install/zip/tomcat.md:104]: https://artifacts.alfresco.com/nexus/#nexus-search;gav~org.alfresco~alfresco-content-services-community*~7.3.0~~

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**